airplane/client.py

```diff
@@ -14,15 +15,46 @@ def __init__(self, api_host, api_token):
         self._api_host = api_host
         self._api_token = api_token
 
+    def set_output(self, value, path=""):
+        """Sets the task output. Optionally takes a JSON path which can be used
+        to set a subpath"""
```

Comment:
minor nit, id like python docstrings to end in newline if multi-line (https://www.python.org/dev/peps/pep-0008/#documentation-strings)

```suggestion
        to set a subpath
        """
```